python/freetensor/core/RPCTools/center.py

```diff
@@ -0,0 +1,102 @@
+from email.header import make_header
+from sre_constants import SUCCESS
+from xmlrpc.server import SimpleXMLRPCServer
+import xmlrpc.client as client
+import sys, socket, uuid, os, time
+import threading
+
+List = {}
```

Comment:
What is the `List`, and why did you name `dict` as `List` :)

Should be a more specific name, in `under_score` fassion.